packages/flutter_tools/test/general.shard/flutter_manifest_test.dart

```diff
@@ -124,29 +146,32 @@ flutter:
         - asset: a/bar
           weight: 400
 ''';
-      final FlutterManifest flutterManifest = FlutterManifest.createFromString(manifest);
-      final dynamic expectedFontsDescriptor = [{'fonts': [{'asset': 'a/bar'}, {'weight': 400, 'asset': 'a/bar'}], 'family': 'foo'}]; // ignore: always_specify_types
```

Comment:
I removed all of the fontsDescriptor expectations and instead made a single test that verifies that the fontsDescriptor is created correctly